vtr_flow/tasks/regression_tests/vtr_reg_basic/hdl_include_yosys_odin/config/config.txt

```diff
@@ -0,0 +1,41 @@
+#################################################################
+# Configuration file for running experiments                    #
+#                                                               #
+# This config file is testing the ability to specify include    #
+# files that should pass to the VTR frontend with the top       #
+# module of the benchmark (ch_intrinsic_top.v). This is done    #
+# by specifying two Verilog header files that provide essential # 
+# definitions, and memory_controller design that provides the   #
```

Comment:
Same comments as review comments above.

Also, it seems this comment block should mention this test is for the yosys+odin front end?